TODO.rst

```diff
@@ -6,6 +6,13 @@ TODO List
 Short-term
 ----------
 
+To be done before release:
```

Comment:
These should be issues, not tracked in the repo IMO.

Reply:
But if this slows down this PR, you can also create an issue to convert `TODO.rst` into issues :-)

Reply:
Followed up in #14.